core/App/localization/fr/index.ts

```diff
@@ -1,98 +1,197 @@
+        "IAgree": "J'accepte",
+        "Attestation": "J'ai lu, je comprends et j'accepte les conditions."
+    },
+    "PinCreate": {
+        "UserAuthenticationPin": "NIP d'authentification de l'utilisateur",
+        "PinMustBe6DigitsInLength": "Le NIP doit comporter 6 chiffres",
+        "PinsEnteredDoNotMatch": "Les NIP saisis ne correspondent pas",
+        "6DigitPin": "NIP à 6 chiffres",
+        "ReenterPin": "Saisir le NIP à nouveau",
+        "Create": "Créer"
+    },
+    "PinEnter": {
+        "IncorrectPin": "NIP incorrect"
+    },
+    "ContactDetails": {
+        "Created": "Créé",
+        "ConnectionState": "État de la connexion",
+        "AContact": "Un contact"
+    },
+    "Credentials": {
+        "CredentialsNotFound": "Justificatifs non trouvés",
+        "CredentialDetails": "Détails des justificatifs"
+    },
+    "CredentialDetails": {
+        "Id": "Identifiant :",
+        "CreatedAt": "Créé à :",
+        "Version": "Version",
+        "Issued": "Délivré",
+        "PrivacyPolicy": "Politique de confidentialité",
+        "TermsAndConditions": "Conditions d'utilisation",
+        "RemoveFromWallet": "Retirer du portefeuille",
+        "Revoked": "Révoqué",
+        "CredentialRevokedMessageTitle": "Ce justificatif est révoqué",
+        "CredentialRevokedMessageBody": "Ce justificatif peut ne plus fonctionner pour certaines demandes de preuve. Vous devrez mettre à jour le justificatif avec l'émetteur."
+    },
+    "Home": {
+        "Welcome": "Bienvenue",
+        "Notifications": "Notifications",
+        "NoNewUpdates": "Vous n'avez pas de nouvelles notifications.",
+        "NoCredentials": "Vous n'avez pas de justificatifs dans votre portefeuille.",
+        "SeeAll": "Voir tout",
+        "YouHave": "Vous avez",
+        "Credential": "justificatif",
+        "Credentials": "justificatifs",
+        "InYourWallet": "dans votre portefeuille"
+    },
+    "Scan": {
+        "SuccessfullyAcceptedConnection": "Connexion acceptée avec succès",
+        "AcceptingConnection": "Acceptation de la connexion",
+        "ConnectionRecordIdNotFound": "Identifiant de l'enregistrement de connexion non trouvé",
+        "ConnectionAccepted": "Connexion acceptée",
+        "ConnectionNotFound": "Connexion non trouvée",
+        "InvalidQrCode": "Code QR non valide. Veuillez réessayer.",
+        "UnableToHandleRedirection": "Impossible de traiter la redirection",
+        "Close": "Fermer",
+        "Torch": "Flash"
+    },
+    "Connection": {
+        "JustAMoment": "Veuillez patienter pendant que nous établissons une connexion sécurisée...",
+        "TakingTooLong": "Cela prend plus de temps que d'habitude. Vous pouvez retourner à l'accueil ou continuer à attendre."
+    },
+    "CredentialOffer": {
+        "ThisIsTakingLongerThanExpected": "Cela prend plus de temps que prévu. Revenez plus tard pour votre nouveau justificatif.",
```

Comment:
Pourquoi il y a 2 tournures de phrase différentes?

- Cela prend plus de temps que d'habitude
-"Cela prend plus de temps que prévu

Reply:
Il s'agit d'une traduction de la version anglaise:

- This is taking longer than usual
- This is taking Longer than expected

Reply:
Si accepté, voir section Loading